### PR TITLE
Centralize transform functions

### PR DESCRIPTION
**Pull request by archmoj**
@plotly/plotly_js

### CODE REVIEW

File: src/lib/index.js [hunk 1]
```diff
@@ -1195,6 +1195,18 @@ lib.isHidden = function(gd) {
     return !display || display === 'none';
 };
 
+lib.strTranslate = function(x, y) {
+    return (x || y) ? 'translate(' + x + ',' + y + ')' : '';
+};
```
Comment (alexcjohnson) [inline src/lib/index.js:1200]: Might be able to simplify even more with an optional 3rd argument which is the number of digits to round to - would remove the `Math.round` and `toFixed` you have in a few places.

File: src/plots/cartesian/axes.js [hunk 2]
```diff
@@ -2867,7 +2882,7 @@ axes.drawLabels = function(gd, ax, opts) {
             } else {
                 var mjWidth = Drawing.bBox(mathjaxGroup.node()).width;
                 var mjShift = mjWidth * {end: -0.5, start: 0.5}[anchor];
-                mathjaxGroup.attr('transform', transform + (mjShift ? 'translate(' + mjShift + ',0)' : ''));
+                mathjaxGroup.attr('transform', transform + (mjShift ? strTranslate(mjShift, 0) : ''));
```
Comment (alexcjohnson) [inline src/plots/cartesian/axes.js:2885]: you don't need `mjShift ?`, `strTranslate` already does that :)

**Review by alexcjohnson — APPROVED** [review]
Nice simplification! 💃

File: src/plots/cartesian/axes.js (hunk 2 above)
Comment (archmoj) [reply src/plots/cartesian/axes.js]: Good call. Addressed in https://github.com/plotly/plotly.js/pull/5254/commits/15c4fa02bde0db678dcd4637a9c228df55c9a6d3.

**Pull request merged**